xcvm/lib/core/src/program.rs

```diff
@@ -0,0 +1,12 @@
+use alloc::vec::Vec;
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+use serde::{Deserialize, Serialize};
+
+#[cfg_attr(feature = "std", derive(schemars::JsonSchema))]
+#[derive(Clone, PartialEq, Eq, Debug, Encode, Decode, TypeInfo, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Program<Instructions> {
+	pub tag: Option<Vec<u8>>,
```

Comment:
it can be just tag: Vec<u8> ? may be. (can be changed later to 256 values bounded vec)